### ISSUE TITLE
Don't show success message if chef-client run fails

### ISSUE THREAD

**Issue by dougireton** [Nordstrom/chefdk_bootstrap#33]
```
Starting Chef Client, version 12.4.1
[2015-08-17T22:30:45-07:00] FATAL: chef-client doesn't have administrator privileges on node my_laptop.example.com.
[2015-08-17T22:30:45-07:00] FATAL: fatal_windows_admin_check is set to TRUE.

Running handlers:
[2015-08-17T22:30:45-07:00] ERROR: Running exception handlers
Running handlers complete
[2015-08-17T22:30:45-07:00] ERROR: Exception handlers complete
Chef Client failed. 0 resources updated in 33.885253 seconds
[2015-08-17T22:30:45-07:00] FATAL: Stacktrace dumped to C:/Users/doug/chef/local-mode-cache/cache/chef-stacktrace.out
[2015-08-17T22:30:45-07:00] FATAL: Chef::Exceptions::WindowsNotAdmin: chef-client doesn't have administrator privileges on node my_laptop.example.com


*Congrats!!! Your workstation is now set up for Chef Development!*
```

**Comment by johndkraft** [comment]
Still an issue on Windows

**Comment by dougireton** [comment]
Closed with v1.3.2
